easybuild/framework/easyblock.py

```diff
@@ -2229,6 +2241,13 @@ def test_step(self):
 
             return out
 
+    def _test_step(self):
+        """Run the test_step and handles failures"""
+        try:
+            self.test_step()
+        except EasyBuildError as e:
```

Comment:
Don't use single-letter variable names outside of list comprehensions please, use `err`?